luarules/gadgets/game_unit_market.lua

```diff
@@ -378,9 +374,7 @@ else -- unsynced
     -- lets only broadcast these trades to allies and spectators
```

Comment:
`spec` is unused, as well as `myPlayerID`, `myAllyTeamID` both here and inside the `PlayerChanged()` callin